client.lua

```diff
@@ -12,12 +12,19 @@ local cam2 = nil
 
 -- Functions
 
-local function SetDisplay(bool)
-    choosingSpawn = bool
-    SetNuiFocus(bool, bool)
+--- Displays the spawn UI and disables controls
+---@param isChoosingSpawn boolean
+local function SetDisplay(isChoosingSpawn)
+    -- launches a thread to disable controls if showing the UI
+    if choosingSpawn then launchDisableControlsThread() end
```

Comment:
Could this be isChoosingSpawn instead?

Reply:
Maybe choosingSpawn should be set here as well as it was being done? Otherwise couldn't this change the behavior?

Reply:
I changed to isChoosingSpawn I think its better for us as well.